modules/op-installing-pipelines-as-code-on-an-openshift-cluster.adoc

```diff
@@ -0,0 +1,60 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="installing-pipelines-as-code-on-an-openshift-cluster_{context}"]
+= Installing Pipelines as Code on an OpenShift cluster
+
+[discrete]
+.Procedure
+
+. To install Pipelines as Code on a {product-title} cluster, run the following command:
++
+[source,terminal]
+----
+$ VERSION=0.5.4
+$ oc apply -f https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/release-$VERSION/release-$VERSION.yaml
+----
++
+[NOTE]
+====
+For the latest stable version, check the link:https://github.com/openshift-pipelines/pipelines-as-code/releases[release page]. In addition, check the {pipelines-title} release notes to ensure that the Pipelines as Code version is compatible with the {pipelines-shortname} version.
+====
++
+Pipelines as Code is installed in the `pipelines-as-code` namespace, along with creation of roles and route URL for the Pipelines as Code controller.
```

Comment:
```suggestion
Pipelines as Code is installed in the `pipelines-as-code` namespace. As part of this installation process, user roles and the route URL for the Pipelines as Code controller are also created in the namespace.
```
This sentence seems awkward to me. Consider breaking it into two sentences as I suggest above.